### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by sileht**
Fixes MRGFY-3944

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # gha-ci-issues
 
+GitHub Action to automatically uploads Junit XML report to Mergify CI Issues
```
Comment (jd) [inline README.md:3]:
```suggestion
GitHub Action to automatically uploads JUnit XML reports to Mergify CI Issues.
```

**Review by jd — APPROVED** [review]

**Review by lecrepont01 — APPROVED** [review]

**Comment by sileht** [comment]
@mergifyio requeue

**Comment by sileht** [comment]
@mergifyio refresh

**Comment by jd** [comment]
@mergify refresh

**Comment by sileht** [comment]
@mergify refresh

**Comment by sileht** [comment]
@mergify refresh

**Comment by Greesb** [comment]
@mergifyio requeue

**Pull request merged**